pkg/clusters/addons/kong/kong.go

```diff
@@ -296,3 +386,69 @@ func urlForService(ctx context.Context, cluster clusters.Cluster, nsn types.Name
 
 	return nil, fmt.Errorf("service %s has not yet been provisoned", service.Name)
 }
+
+// deployKongEnterpriseLicenseSecret deploy secret using license json data
+func deployKongEnterpriseLicenseSecret(ctx context.Context, cluster clusters.Cluster, namespace, name, licenseJSON string) error {
+	newSecret := &corev1.Secret{
+		Type: corev1.SecretTypeOpaque,
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Data: map[string][]byte{
+			"license": []byte(licenseJSON),
+		},
+	}
+
+	_, err := cluster.Client().CoreV1().Secrets(namespace).Create(ctx, newSecret, metav1.CreateOptions{})
+	if err != nil {
+		return fmt.Errorf("failed creating kong-enterprise-license secret, err %w", err)
+	}
+
+	fmt.Printf("successfully deployed kong-enterprise-license into the cluster .")
```

Comment:
we shouldn't print to stdout in library code:

```suggestion
```

Reply:
eed83b98890e9fc320e3f79e00d47c6439734dba